chsdi/views/printmulti.py

```diff
@@ -145,7 +145,11 @@ def _qrcodeurlparse(raw_url):
 def _qrcodeurlunparse(url_tuple):
     (qrcode_service_url, map_url, params) = url_tuple
 
-    quoted_map_url = quote_plus(map_url + "?url=" + unquote_plus(urlencode(params)))
+    str_params = {}
+    for k, v in params.iteritems():
+            str_params[k] = unicode(v).encode('utf-8')
```

Comment:
What about :
`params = dict([(k, unicode(v).encode('utf-8')) for k, v in params.items()])`
or
`params = dict(map(lambda x: (x[0], unicode(x[1]).encode('utf-8')), params.items()))`